app/languages/definitions/Catalan.yml

```diff
@@ -0,0 +1,14 @@
+locale: ca-ES
+dictionaryFile: ca-ES-utf8.csv
+name: Català
```

Comment:
TT9 knows how to generate the name by using the locale. Specifying this property is only necessary, if it appears incorrectly in the language list. Please, try without a `name`.